app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
```

Comment:
If something goes wrong, how is the user supposed to understand what happened?
At least add some informative `print` statement here.